modules/cvv/src/qtutil/filter/diffFilterWidget.cpp

```diff
@@ -69,8 +69,8 @@ void DiffFilterFunction::applyFilter(InputArray in, OutputArray out) const
 	}
 
 	cv::Mat originalHSV, filteredHSV;
-	cv::cvtColor(in.at(0).get(), originalHSV, COLOR_BGR2HSV);
-	cv::cvtColor(in.at(1).get(), filteredHSV, COLOR_BGR2HSV);
```

Comment:
What is used version of OpenCV?

`using namespace cv;` on line 16 should fix that properly.